.travis.yml

```diff
@@ -41,7 +41,7 @@ matrix:
 
 env:
   global:
-    - HTSDIR=./htslib
+    - HTSDIR=./hidden-htslib
```

Comment:
If you made this `../hidden-htslib` you wouldn't have to change `check_copyright.pl` or `check_spaces.pl`.

Reply:
True, but it doesn't matter either way frankly.  The change isn't exactly an onerous one and I don't like the idea of trying to escape out of the directory structure provided for me by travis.